src/handlers/web3.ts

```diff
@@ -698,53 +703,51 @@ export const getDataForNftTransfer = async (
   from: string,
   to: string,
   asset: ParsedAddressAsset
-): Promise<string> => {
-  const nftVersion = asset.asset_contract?.nft_version;
-  const schema_name = asset.asset_contract?.schema_name;
-  if (nftVersion === '3.0') {
-    const transferMethodHash = smartContractMethods.nft_transfer_from.hash;
-    const data = ethereumUtils.getDataString(transferMethodHash, [
-      ethereumUtils.removeHexPrefix(from),
+): Promise<string | undefined> => {
+  if (!asset.id || !asset.asset_contract?.address) return;
+  const lowercasedContractAddress = asset.asset_contract.address.toLowerCase();
+  const standard = asset.asset_contract?.schema_name;
+  let data;
+  if (
```

Comment:
crypto kitties and crypto punks are ancient nfts. they were actually created before the erc721 standard was finalized. as a result, the contract methods used for transfers of these nfts are different from what the erc721 standard defines today. these edge cases are dealt with here. if we encounter other popular nft collections that don't adhere to erc721, we should add support here.